app/src/ui/rebase/progress-dialog.tsx

```diff
@@ -30,21 +22,17 @@ export class RebaseProgressDialog extends React.Component<
     // this dialog is undismissable, but I need to handle the event
   }
 
-  /** After a delay, run the assigned action to start/continue the rebase */
-  public async componentDidMount() {
-    if (this.props.rebaseAction) {
-      await timeout(500)
-      await this.props.rebaseAction()
-    }
-  }
-
   public render() {
     const {
       rebasedCommitCount,
-      commits,
+      totalCommitCount,
       value,
       currentCommitSummary,
     } = this.props.progress
+
+    // ensure progress always starts from 0
+    const count = rebasedCommitCount === 0 ? 1 : rebasedCommitCount
```

Comment:
It's not clear how this variable or the logic relates to the comment above. It seems that it's possible to set this to some non-zero value.

Reply:
Improved in 912623da75bd6dff5e81e4cf672ef035b3e787d7